pages/common/bandwhich.md

```diff
@@ -0,0 +1,24 @@
+# bandwhich
+
+> A CLI utility for displaying current network utilization by process, connection and remote IP/hostname.
+> More information: <https://github.com/imsnif/bandwhich>.
+
+- Show remote addresses table only:
+
+`bandwhich --addresses`
+
+- Show DNS queries:
+
+`bandwhich --show-dns`
+
+- Show total (cumulative) usages:
+
+`bandwhich --total-utilization`
+
+- Show network utilization for given interface (eg. {{eth0}}):
+
+`bandwhich --interface {{interface}}`
+
+- Show DNS queries with a given DNS server:
+
+`bandwhich --show-dns --dns-server {{dns-server}}`
```

Comment:
```suggestion
- Show DNS queries for a specific DNS server:

`bandwhich --show-dns --dns-server {{dns_server_ip}}`
```

Reply:
@bl-ue It doesn't show DNS quiries _"for_" a specific DNS server, it uses the given DNS server.

Reply:
(sorry, forgot about this one) Updated.  Is it correct now?

Reply:
I'm a bit 50-50; it *uses* the specified DNS server, your sentence makes it seem like doing queries to the DNS server. It's a bit, "how you read it, then it makes sense"...

Reply:
Ah, I see @Waples. What about this?

```suggestion
- Show DNS queries with a given DNS server:

`bandwhich --show-dns --dns-server {{dns_server_ip}}`
```